wsgi/kuma.wsgi

```diff
@@ -29,6 +29,10 @@ import manage
 from django.core.wsgi import get_wsgi_application
 django_app = get_wsgi_application()
 
+# Enable WhiteNoise
+from whitenoise.django import DjangoWhiteNoise
+django_app = DjangoWhiteNoise(django_app)
```

Comment:
There are  a few options we can set here, e.g. subclassing the class and modifying the `EXTRA_MIMETYPES` value may make sense to support more of those we have in the Apache config: https://github.com/evansd/whitenoise/blob/b3974c2ea80e93ba086331cff447a09a427a1a9f/whitenoise/base.py#L72-L73